### DISCUSSION TITLE
beginer learning

### DISCUSSION THREAD

**Discussion by rjngh2005** [nextgenhealthcare/connect#4659] [General]
I am a beginner 
Please guide me how to learn mirth

**Comment by pacmano1** [comment]
I would suggest you start by googling "learning Mirth".   Review YouTube Videos.   Have an understanding of the transports and protocols for your exact uses cases, e.g. hl7, APIs, etc.   Take the formal Mirth training taught by Nextgen.   Make sure you have a good understanding of Javascript (in particular the Rhino flavor).   Go over to the mirth forums and read threads.

Your question is much like "How do I build a car?" - in other words your scope is very broad and it's hard to provide specific help.

Your approach to learning mirth should likely be not all that different than your approach to learning anything else.

**Reply by tonygermano** [reply]
I'd add download and read the user guide to that list.

**Comment by pacmano1** [comment]
And perhaps get http://mirthconnect.shamilpublishing.com/.  Also: https://www.amazon.com/Up-Speed-Mirth-Connect-NextGen/dp/B08JF5FGTG/ref=sr_1_1?dchild=1&keywords=learning+mirth+connect&qid=1627995661&sr=8-1

I have not read either though.

**Reply by rjngh2005** [reply]
I guess my question is - which videos are good on youtube 
since there are more than 50-60 videos please suggest a few good ones

**Reply by pacmano1** [reply]
Watch them all.    Then please post back here with what you found the most helpful.

**Reply by rjngh2005** [reply]
anyone else can suggest ?

**Reply by tonygermano** [reply]
I'll reiterate what I said above. Download and read the user guide. People are happy to help you with specific questions, but a single discussion post or a few videos on isolated topics are not going to be a substitute for the books and classes already suggested in the comments.

Once you have spent some time playing with the software and have an idea of something you want to do with it, come back if you need help.

**Reply by twest-mirthconnect** [reply]
I would attend class. There's a great opportunity to get a lot done there during class, through labs, discussion groups, etc.

**Comment by rjngh2005** [comment]
can u send me more details
a) cost
b) schedule
c) link to more details of the syllabus

On Tue, Aug 24, 2021 at 2:20 PM Travis West ***@***.***>
wrote:

> I would attend class. There's a great opportunity to get a lot done there
> during class, through labs, discussion groups, etc.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nextgenhealthcare/connect/discussions/4659#discussioncomment-1229816>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AN35WTOWXJVBHGZ7TGY3Z2TT6QEJNANCNFSM5BNZUH5A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&utm_campaign=notification-email>
> .
>


-- 
Rajan Gehi
I love Chicken Shawarma
http://en.wikipedia.org/wiki/Shawarma

**Reply by pacmano1** [reply]
Did you look at https://www.nextgen.com/products-and-services/connect-training?  Some of the information is there.  Not all though.

**Reply by twest-mirthconnect** [reply]
If you send me your contact information, I will have someone give you a call to talk a bit more about it in addition to what @pacmano1 shared.

twest@nextgen.com
